generator/templates/error_handler.go.tmpl

```diff
@@ -4,6 +4,27 @@
 {{- $grpcErrors := $def.GRPCErrors }}
 {{- if $grpcErrors }}
 grpcfed.SetGRPCError(ctx, value, err)
+var (
+	defaultMsg string
+	defaultCode grpcfed.Code
+	defaultDetails []grpcfed.ProtoMessage
+)
+if stat, exists := grpcfed.GRPCStatusFromError(err); exists {
+	defaultMsg = stat.Message()
+	defaultCode = stat.Code()
+	details := stat.Details()
+	defaultDetails = make([]grpcfed.ProtoMessage, 0, len(details))
+	for _, detail := range details {
+		msg, ok := detail.(grpcfed.ProtoMessage)
+		if ok {
+			defaultDetails = append(defaultDetails, msg)
+		}
+	}
+	_ = defaultMsg
+	_ = defaultCode
+	_ = defaultDetails
```

Comment:
What is this for?

Reply:
To prevent build errors for unused variables